src/controllers/pinController.ts

```diff
@@ -711,6 +752,7 @@ export class PINController extends Controller {
      */
     private async createOrRecreatePinServiceBC(
         requestBody: serviceBCCreateRequestBody,
+        regenerateOrCreate: string,
```

Comment:
Only problem with this is that you'll have to update the FE to actually use the correct endpoint, and we have to assume that whatever has been sent is correct... you may get a regenerate email when you were actually creating and vice versa

Reply:
oooooh good point, let me fix that